app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/edititem/EditItemScreen.kt

```diff
@@ -381,7 +386,7 @@ private fun AdvancedOptions(
                     BitwardenMultiSelectButton(
                         modifier = Modifier
                             .fillMaxSize()
-                            .semantics { testTag = "ItemTypePicker" },
+                            .semantics { testTag = "OTPTypeItemTypePicker" },
```

Comment:
⛏️ Just want to double check this is the correct tag string. Seems like it may be a typo? `OTPItemTypePicker` reads a little bit better.

Reply:
good catch! agreed this was a typo - pushed a changed